ibis/backends/snowflake/registry.py

```diff
@@ -181,28 +187,16 @@ def _extract_url_query(t, op):
     # ibis.expr.operations.maps
     ops.MapKeys,
     # ibis.expr.operations.reductions
-    ops.All,
-    ops.Any,
     ops.ArrayCollect,
-    ops.BitAnd,
-    ops.BitOr,
-    ops.BitXor,
     ops.MultiQuantile,
-    ops.NotAll,
-    ops.NotAny,
     # ibis.expr.operations.strings
     ops.FindInSet,
-    ops.RegexExtract,
-    ops.RegexReplace,
-    ops.RegexSearch,
     ops.StringSplit,
     # ibis.expr.operations.structs
     ops.StructField,
     # ibis.expr.operations.temporal
-    ops.DateFromYMD,
     ops.ExtractMillisecond,
```

Comment:
To implement this function, you have to copy the code from Postgres and change milliseconds to `epoch_mliseconds`.

Reply:
https://docs.snowflake.com/en/sql-reference/functions-date-time.html#supported-date-and-time-parts

Reply:
Done, thanks!